modules/core/src/main/java/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -426,6 +426,8 @@ public String handleResponse(final ClassicHttpResponse response) throws IOExcept
                     long fileSize = fileItem.getSize();
 
                     File file = allocateFile(rawDataDir, fileName);
+                    String bestFormat = ImportingManager.extensionToFormat.get(file.getName().substring(file.getName().indexOf('.')));
```

Comment:
This will fail for filenames which don't have any `.` in them. Instead of re-implementing this, we should rather go through `ImportingManager.getFormat` which already contains the required logic. See my comment above.

Reply:
Updated to use ImportingManger.getFormatFromFileName method